### PR TITLE
Custom frame axes options: scale, dashed lines and no pan&zoom frame

### PR DESCRIPTION
**Pull request by viktorku**
Adds an ability to optionally specify:

- Custom `Axes` and `OrbitControl`'s "temporary" pan and zoom frame line type to `"dashed"` and custom scale, as it would be nice if we display some frames with a different scale/line-type (besides the thicknes - which we can already do customize).
- Also the "temporary" pan and zoom frame can be omitted from being drawn, as sometimes it might just unnecessary clutter the view.

Backwards compatibility with default values is being retained, but please confirm.

image

Credits go to @sdeleersnijder ;)

### CODE REVIEW

File: src/models/Axes.js [hunk 1]
```diff
@@ -53,12 +65,29 @@ ROS3D.Axes = function(options) {
...
+    var line;
+    if (lineType === 'dashed') {
+      var l = lineDashLength;
+      for (var i = 0; (l / 2 + 3 * l * i + l / 2) <= 1; ++i) {
+        var geom = new THREE.CylinderGeometry(shaftRadius, shaftRadius, l);
+        line = new THREE.Mesh(geom, material);
+        line.position.copy(axis);
+        // Make spacing between dashes equal to 1.5 times the dash length.
+        line.position.multiplyScalar(l / 2 + 3 * l * i);
+        line.quaternion.copy(rot);
```
Comment (T045T) [inline src/models/Axes.js]: There's a few duplicated lines here - can you refactor the code to move them outside the if-else?

Comment (viktorku) [reply src/models/Axes.js]: They are not duplicated, but inside a for loop. The de-duplication would make reasoning about this harder.

File: src/visualization/interaction/OrbitControls.js [hunk 2]
```diff
@@ -61,17 +69,19 @@ ROS3D.OrbitControls = function(options) {
   };
   var state = STATE.NONE;
 
-  // add the axes for the main coordinate frame
   this.axes = new ROS3D.Axes({
```
Comment (jihoonl) [inline src/visualization/interaction/OrbitControls.js]: `this.axes` isn't used if `displayPanAndZoomFrame` is `false`. Could you move this under if statement? And update `showAxes()` something like
```
 if(this.displayPanAndZoomFrame){
 var that = this;                                                                                                                                                                                                                                                           
  this.axes.traverse(function(obj) {                                                                                                   
    obj.visible = true;                                                                                                                
  });                                                                                                                                  
  if (this.hideTimeout) {                                                                                                              
    clearTimeout(this.hideTimeout);                                                                                                    
  }                                                                                                                                    
  this.hideTimeout = setTimeout(function() {                                                                                           
    that.axes.traverse(function(obj) {                                                                                                 
      obj.visible = false;                                                                                                             
    });                                                                                                                                
    that.hideTimeout = false;                                                                                                          
  }, 1000);                                                                                                                            
}
```

Comment (jihoonl) [reply src/visualization/interaction/OrbitControls.js]: And [line490-492](https://github.com/RobotWebTools/ros3djs/blob/develop/src/visualization/interaction/OrbitControls.js#L490)

**Comment by jihoonl** [comment]
It is an interesting feature that I would like to merge in. @viktorku any updates?

**Pull request merged**